MB-3552: Removing prd safeties from upload-secure-migrations

cblkwell opened the pull request:
## Description

This switches on the functionality for uploading migrations into the bucket for the prd environment. I've added the appropriate permissions for engineers in the GovCloud `prd` account and tested to make sure they can do an `ls` of the bucket and `cp` the files appropriately. I am not sure if we want to merge this immediately, but this should close the loop and make it possible to move forward with `prd`.

robot-mymove commented:
<!--
  0 failure: 
  0 warning: 
  1 messages
  
  DangerID: danger-id-Danger;
-->



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>:link: <a href="https://dp3.atlassian.net/browse/MB-3552">MB-3552</a></td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9a766458bcc5170ff76b1b879f7e2252a699d03f
</p>

cblkwell commented:
> Please also change these lines that talk about this only working in exp/stg for govcloud: https://github.com/transcom/mymove/blob/master/Makefile#L855
> https://github.com/transcom/mymove/blob/master/scripts/generate-secure-migration#L77
> https://github.com/transcom/mymove/blob/master/scripts/pricing-import#L126
> https://github.com/transcom/mymove/blob/master/scripts/README.md
> 
> I think these should be in the same pr as this.

Hrm. Do engineers need to be able to *get* the migrations from the prod bucket as well? I didn't realize that. :/

kilbergr commented:
@cblkwell currently they do retrieve them from prod in commercial. There's a ticket to address pulling them from govcloud here: https://dp3.atlassian.net/browse/MB-3639
I don't think you have to do that work here.

cblkwell commented:
> https://dp3.atlassian.net/browse/MB-3639

Yeah, I updated the permissions. I've added the things I think you wanted me to fix -- I also made some other changes to the `Makefile` that I'm not sure you wanted, but seemed like it was a good idea, like making the `run_$ENV_migrations` targets run against both commercial and AWS -- if you think that's a bad idea I can revert those particular changes.

kilbergr commented on Makefile:
```diff
@@ -852,20 +852,19 @@ run_com_prod_migrations: bin/milmove db_deployed_migrations_reset ## Run Commerc
 .PHONY: run_staging_migrations
-run_staging_migrations: run_com_staging_migrations ## Currently: Run Commercial Staging migrations against Deployed Migrations DB in commercial
-	# run_gov_staging_migrations
+run_staging_migrations: run_com_staging_migrations i
```
```suggestion
run_staging_migrations: run_com_staging_migrations
```
